includes/login-register.php

```diff
@@ -325,7 +325,9 @@ function give_email_access_login() {
 
 	$recaptcha_key    = give_get_option( 'recaptcha_key' );
 	$recaptcha_secret = give_get_option( 'recaptcha_secret' );
-	$enable_recaptcha = ! empty( $recaptcha_key ) && ! empty( $recaptcha_secret ) ? true : false;
+	$enable_recaptcha = give_get_option( 'enable_recaptcha' );
+
+	$enable_recaptcha = ( 'on' === $enable_recaptcha ) && ! empty( $recaptcha_key ) && ! empty( $recaptcha_secret ) ? true : false;
```

Comment:
Kindly used `give_is_setting_enabled` function to check if reCAPTCHA is Enable or not